pkg/client/localsubjectaccessreview.go

```diff
@@ -63,11 +64,12 @@ func (c *localSubjectAccessReviews) Create(sar *authorizationapi.LocalSubjectAcc
 		}
 		deprecatedResponse := &authorizationapi.SubjectAccessReviewResponse{}
 
-		req, err := overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
+		var deprecatedReq *restclient.Request
```

Comment:
yes, it is to avoid using `:=` to initialize the vars, and so shadow the `err` var, which could then be set to `nil`